test/e2e/network_create_test.go

```diff
@@ -427,15 +428,19 @@ var _ = Describe("Podman network create", func() {
 	})
 
 	It("podman network create with invalid IP", func() {
-		nc := podmanTest.Podman([]string{"network", "create", "--subnet", "10.11.0/17000", stringid.GenerateRandomID()})
```

Comment:
Another test that was completely broken. PLEASE REVIEW THIS CAREFULLY.

Reply:
chnage looks good but you may want to rename the test to include ip-range

Reply:
Before I do that, is it possible that I misunderstood the point of the test? What would you expect "invalid IP" to mean in the context of "network create"??

Reply:
My initial gut was running a container with --ip that is not part of the network subnet but AFAIK we already have tests for that. I also see that we handle invalid subnet before in a different test, so that leaves --gateway which also accepts a ip.